sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Repositories/ContainerRepositoryClient.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Containers.ContainerRegistry
+{
+    /// <summary> The repository service client. </summary>
+    public partial class ContainerRepositoryClient
+    {
+        private readonly HttpPipeline _pipeline;
+        private readonly ClientDiagnostics _clientDiagnostics;
+        private readonly RepositoryRestClient _restClient;
+
+        private string _repository;
```

Comment:
Is this one intentionally not marked as `readonly`? It appears to only be set in the ctor.

Reply:
No, I will mark it as such.  Thanks!